php/libraries/Utility.class.inc

```diff
@@ -574,73 +555,6 @@ class Utility
         return $var;
     }
 
-    /**
-     * Explodes an encapsulted, delimited string like
-     * 'hello','my name is \'bob\'', 'etc'
-     *
-     * @param string $seperator unknown
-     * @param string $delimiter unknown
-     * @param string $string    unknown
-     *
-     * @return unknown
-     *
-     * @note    Don't know what the above function comment is supposed to mean,
-     *       so just preserving it for now. Appears to only be used
-     *       by NDB_Form_candidate_parameters and if so, should be moved there.
-     * @cleanup
-     */
-    static function explodeEncapsulated($seperator, $delimiter, $string)
```

Comment:
used in candidate_parameters on IBIS

@sruthymathew123 if you can get rid of it or move it to the class it is used in, then when you confirm we'll be able to remove safely